x/auth/ante_test.go

```diff
@@ -12,6 +12,9 @@ import (
 	wire "github.com/cosmos/cosmos-sdk/wire"
 )
 
+func nopFeeHandler(ctx sdk.Context, tx sdk.Tx, fee sdk.Coins) {
```

Comment:
what does `nop` stand for? Why not call it `BurnFeeHandler` and have the apps and other examples import it?

Reply:
Good idea, renamed `BurnFeeHandler`, basecoin & democoin now import